ui/package.json

```diff
@@ -58,6 +58,7 @@
   },
   "dependencies": {
     "@types/chartjs": "0.0.31",
-    "c3": "^0.7.20"
+    "c3": "^0.7.20",
```

Comment:
Are you ready to remove the dependency on C3?

Reply:
I hadn't removed the AppChartC3 yet just because I still switch back to that chart sometimes to test